release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2782,6 +2782,11 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |General Availability
 
+|NUMA Resources Operator
```

Comment:
I think the best spot for this is table 22 `Scalability and performance`

Reply:
Thanks, I've moved it.